[FSTORE-1095] update driver dependecies and secrets feature (#350)
logicalclocks/logicalclocks.github.io: docs/user_guides/fs/storage_connector/creation/jdbc.md

Enter the details for your JDBC enabled database.

<figure markdown>
  ![JDBC Connector Creation](../../../../assets/images/guides/fs/storage_connector/jdbc_creation.png)
  <figcaption>JDBC Connector Creation Form</figcaption>

> </figure>

Reviewer: maybe we could just suffice with 1 image?

Author: fixed

1. Select "JDBC" as connector protocol.
2. Enter the JDBC connection url. This can for example also contain the username and password.

> 6. Add driver class name as an argument. This is a mandatory argument even if using default MySQL driver. Add an argument with name `driver` and driver class name as value. For MySQL databases, the class name is `com.mysql.cj.jdbc.Driver`, as shown in example below. The driver class name will differ based on the database to connect.

Reviewer: i dont think this step should be nr. 6

Reviewer: Maybe it should be as a sub-step of nr. 3? since it is about arguments. Or it could be a note.

Author: changed it to a  note

> 3. Add additional key/value arguments to be passed to the connection. These might differ by database. It can be the username and password. Passwords are encrypted as [Secret](../../../projects/secrets/create_secret.md) and stored in the database for additional security.

Reviewer: `Passwords are encrypted as [Secret](../../../projects/secrets/create_secret.md) and stored in the database for additional security.`

it could be a note.

4. Click "Setup storage connector".

<figure markdown>
  ![Adding Driver in arguments](../../../../assets/images/guides/fs/storage_connector/jdbc_driver.png)
  <figcaption>Specifying driver name in case of MySQL</figcaption>
</figure>

!!! note

>   To be able to use the connector, you still need to upload the driver JAR file to the  [Jupyter configuration](../../../projects/jupyter/spark_notebook.md) or to a [Job configuration](../../../projects/jobs/pyspark_job.md)  in `Addtional Jars`.  However, for MySQL connections the default JDBC driver is included in Hopsworks so can skip this step when using MySQL databases.  

Reviewer: ```suggestion
  To be able to use the connector, you need to upload the driver JAR file to the  [Jupyter configuration](../../../projects/jupyter/spark_notebook.md) or to a [Job configuration](../../../projects/jobs/pyspark_job.md)  in `Addtional Jars`.  However, for MySQL connections the default JDBC driver is included in Hopsworks so this step can skipped.
```

Enter the details for your JDBC enabled database.

<figure markdown>
  ![JDBC Connector Creation](../../../../assets/images/guides/fs/storage_connector/jdbc_creation.png)
  <figcaption>JDBC Connector Creation Form</figcaption>
</figure>

1. Select "JDBC" as connector protocol.
2. Enter the JDBC connection url. This can for example also contain the username and password.

> 3. Add additional key/value arguments to be passed to the connection. These might differ by database. It can be the username and password. 

Reviewer: ```suggestion
3. Add additional key/value arguments to be passed to the connection. These might differ by database.
```

    !!! note 

>     Driver class name is a mandatory argument even if using the default MySQL driver. Add it by specifying a property with the name `driver` and class name as value. The driver class name will differ based on the database. For MySQL databases, the class name is `com.mysql.cj.jdbc.Driver`, as shown in the example image.

Reviewer 2: also the formatting is wrong, note should be indented twice

>     Add driver class name as an argument. This is a mandatory argument even if using default MySQL driver. Add an argument with name `driver` and class name as value. The driver class name will differ based on the database to connect. For MySQL databases, the class name is `com.mysql.cj.jdbc.Driver`, as shown in example image.

Reviewer: ```suggestion
    Driver class name is a mandatory argument even if using the default MySQL driver. Add it by specifying a property with the name `driver` and class name as value. The driver class name will differ based on the database. For MySQL databases, the class name is `com.mysql.cj.jdbc.Driver`, as shown in the example image.
```

4. Click "Setup storage connector".

!!! note

> To be able to use the connector, you need to upload the driver JAR file to the [Jupyter configuration](../../../projects/jupyter/spark_notebook.md) or [Job configuration](../../../projects/jobs/pyspark_job.md) in `Addtional Jars`.  For MySQL connections the default JDBC driver is already included in Hopsworks so this step can be skipped.

Reviewer 2: Formatting here is wrong, notes content should be indented twice

> To be able to use the connector, you need to upload the driver JAR file to the  [Jupyter configuration](../../../projects/jupyter/spark_notebook.md) or to a [Job configuration](../../../projects/jobs/pyspark_job.md)  in `Addtional Jars`.  However, for MySQL connections the default JDBC driver is included in Hopsworks so this step can skipped.

Reviewer: ```suggestion
To be able to use the connector, you need to upload the driver JAR file to the [Jupyter configuration](../../../projects/jupyter/spark_notebook.md) or [Job configuration](../../../projects/jobs/pyspark_job.md) in `Addtional Jars`.  For MySQL connections the default JDBC driver is already included in Hopsworks so this step can be skipped.
```
